allure-generator/src/main/javascript/plugins/testresult-severity/SeverityView.js

```diff
@@ -3,12 +3,20 @@ import { escapeExpression } from "handlebars/runtime";
 import { className } from "../../decorators/index";
 import translate from "../../helpers/t";
 
+const severities = ["blocker", "critical", "normal", "minor", "trivial"];
+
 @className("pane__section")
 class SeverityView extends View {
-  template(data) {
-    return data.severity
-      ? `${translate("testResult.severity.name")}: ${escapeExpression(data.severity)}`
-      : "";
+  template({ severity }) {
+    if (!severity) {
+      return "";
+    }
+
+    if (severities.indexOf(severity) >= 0) {
+      return `${translate("testResult.severity.name")}: ${translate(`testResult.severity.${severity}`)}`;
+    }
+
+    return `${translate("testResult.severity.name")}: ${escapeExpression(severity)}`;
```

Comment:
Actually, I am not sure here – do you allow custom values for real? I think that it must be an undocumented feature, then. 🤔

Reply:
@baev do you know about this question maybe?

Reply:
According to [generator](https://github.com/allure-framework/allure2/blob/main/allure-generator/src/main/java/io/qameta/allure/severity/SeverityPlugin.java#L71), it is strictly typed (enum), so we should not expect custom values here.

Reply:
Okay, then, I can simplify the check. Done in 3b9cf370472da09b5a3d3d3ec525fd253e3025fa